### DISCUSSION TITLE
Speaker bureau?

### DISCUSSION THREAD

**Discussion by gavincampbell** [dataplat/DataSaturdays#74] [Ideas]
There have been a few attempts at this in the past. 

Whilst the Saturday events are often in a position of having more submissions than slots, the opposite is true of many evening meetups, and it is possible that some Saturday events might want to "curate" content. 

Is there an opportunity for this site to provide a central contact point for people who might want to speak at events, maybe with some details of topics available etc?

**Comment by amtwo** [comment]
There is definitely a need to support calls for speakers for events. Daniel Hutmacher created a "Call for Data Speakers" resource for the community ([twitter](https://twitter.com/dataspeakers) | [website](https://callfordataspeakers.com/)). 

It's a fairly straightforward mini site that lets you sign up as a speaker to be notified of events + list events for which you need speakers. 

I feel like we could work with Daniel to create seamless integration with his existing service. 

His service is currently (technically) run as part of his business entity so that the company is paying all the expenses--but it's done as a free, "no strings" service to the community. 


Similarly, there's a (new) feature from Sessionize that lets speakers include technologies in their speaker profile, which should help to the same end. 

I think if Data Saturday can integrate with both, it will help coalesce the speaker audience to be registered in both places, and thus make it easier for UG organizers to also leverage those lists (with or without the Data Saturday platform)

**Comment by way0utwest** [comment]
I would agree that we ought to work with Daniel or sessionize here. We could pull data from an API and use it on the site or in email notifications, but those services handle many of those issues well.
